index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="stylesheet" href="style.css">
+	<title>Gleaming Gemstones</title>
+
+</head>
+
+<body>
+	<!--hero section-->
+	<div class="hero-video">
+		<video autoplay muted loop>
+			<source src="3633003-hd_1906_1080_30fps.mp4" type="video/mp4">
+			Din webbläsare stödjer inte HTML5 video.
+		</video>
+
+		<!--text in hero video-->
+		<div class="hero-content">
+			<h1>Gleaming <br>Gemstones</h1>
+		</div>
+
+		<!--navigation menu-->
+		<div class="navbar">
+			<ul>
+				<a href="#e-shop">E-shop</a>
+				<a href="#about">About</a>
+				<a href="#contact">Contact</a>
+			</ul>
+		</div>
+
+		<!--text section-->
+	</div>
+	<h2>Welcome to the World of Gleaming Gemstones</h2>
+	<p>Discover a universe of brilliance and elegance. Subscribe to our newsletter and be the first to explore our
+		dazzling new collections, enjoy exclusive offers, and get expert tips on how to make your gems shine even
+		brighter. Plus, as a thank you, we’ll gift you a 10% discount on your first purchase when you sign up!
+	</p>
+
+	<!--form-->
+	<form action="https://httpbin.org/anything" method="post">
+		<h3>Sign me up!</h3>
+
+		<!--first name-->
+		<label>
+			<input type="text" id="first name" name="first name" placeholder="First name" required><br><br>
+		</label>
+
+		<!--last name-->
+		<label>
+			<input type="text" id="last name" name=" last name" placeholder="Last name" required><br><br>
+		</label>
+
+		<!-- email -->
+		<label>
+			<input type="email" id="email" name="email" placeholder="E-mail" required><br><br>
+		</label>
+
+		<!-- date of birth -->
```

Comment:
smart with date of birth! could be good also to have a required on date of birth as well